src/style.css

```
@@ -1 +1,72 @@
-/* Styles go here */
+html {
+  display: block;
+}
+
+body {
+  display: block;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(143, 10, 10, 0.05);
+  align-items: center;
```

```suggestion
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
  background: #fff;
  box-shadow: 0 2px 4px rgba(143, 10, 10, 0.05);
  
```
Pay attention to the properties order